clients/fides-js/src/components/tcf/TcfOverlay.tsx

```diff
@@ -104,17 +147,21 @@ const transformTcfModelToTcfSave = ({
     return {
       id: model.id,
       preference,
+      served_notice_history_id: noticeSubMap[String(model.id)],
```

Comment:
ah I guess this type is still called `notice` in the backend 😅 ok I take back what I said about distinguishing between notices and TCF...